packages/webapp/src/components/Forms/SimpleCustomType/constants.js

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const CUSTOM_DESCRIPTION = 'custom_description';
+export const DESCRIPTION_MAX_CHARS = 256;
```

Comment:
```suggestion
export const DESCRIPTION_MAX_CHARS = 255;
```

Submitting 256 chars produced this 400 response for me:

![Screenshot 2023-10-26 at 3 35 59 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/9c7afb3f-13da-48e5-b498-cb85e3940cf2)

And I got this pasting the same 256 char string into my db directly:

![Screenshot 2023-10-26 at 3 13 37 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/0daeee80-e2e0-4f04-8152-819b6b59fd0d)

Not sure where db got that 255 value though, I see it wasn't in the migration.

Reply:
Will do; I'll follow-up on the length if you don't get to connect with Loïc before end of day. With the move to textarea half of the main UX issue with the longer field is negated, and I agree with the value of easy-to-pick-out in a list.

Reply:
Thanks, appreciate that! -- such a minor detail but its fun to think about haha